extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -160,6 +161,12 @@ class AmpAccordion extends AMP.BaseElement {
 
       header.addEventListener('click', this.clickHandler_.bind(this));
       header.addEventListener('keydown', this.keyDownHandler_.bind(this));
+      content.addEventListener('rendersubtreeactivation', event => {
+        const section = event.target.closest('amp-accordion > section');
```

Comment:
This is probably too simplistic for our needs. Not sure if we have an array of sections already handy here. If we do - we could just add listeners directly to those elements.

Reply:
It's not clear here since the block is rather large, however this is in the `forEach` block iterating through `this.sections_`.

Reply:
:) In that case let's just use the top `section` var?